sdk/data/azcosmos/cosmos_global_endpoint_manager.go

```diff
@@ -115,19 +117,39 @@ func (gem *globalEndpointManager) GetAccountProperties(ctx context.Context) (acc
 		return accountProperties{}, fmt.Errorf("failed to generate path for name-based request: %v", err)
 	}
 
-	ctx, cancel := context.WithTimeout(ctx, 60*time.Second)
-	azResponse, err := gem.client.sendGetRequest(path, ctx, operationContext, nil, nil)
-	cancel()
+	finalURL := gem.clientEndpoint
+	if path != "" {
+		finalURL = azruntime.JoinPaths(gem.clientEndpoint, path)
+	}
```

Comment:
we don't technically need this code at all (nor lines 113/115-116/118 above) since the request path just ends up being the database account link (which is the default endpoint) - I left it for uniformity in how we create the requests but wanted to point this out to see if we should just remove it

Reply:
I would just remove it